scripts/cybershake/queue_monitor.py

```diff
@@ -29,6 +30,18 @@
 keepAlive = True
 
 
+def send_alert(msg, alert_url):
+    data = {"text": msg}
+
+    req = urllib.request.Request(alert_url)
+    # create header and data, slack-bot requires json data
+    req.add_header("Content-Type", "application/json; charset=utf-8")
+    jsondata = json.dumps(data)
+    jsondataasbytes = jsondata.encode("utf-8")
+    req.add_header("Content-Length", len(jsondataasbytes))
+    response = urllib.request.urlopen(req, jsondataasbytes)
```

Comment:
I think this should be moved to its own file, but it's ok for now. 
If we get any more internet communication stuff it should be moved in future.